pkgs/by-name/ya/yao-pkg/package.nix

```diff
@@ -0,0 +1,106 @@
+{
+  autoPatchelfHook,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  fixup-yarn-lock,
+  lib,
```

Comment:
Nit: customarily lib is placed first, then stdenv

Reply:
Thanks for letting me know on this one, I've applied this change. Good for me to note!